app/src/main/res/values-hi/strings.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2" xmlns:tools="http://schemas.android.com/tools">
     <string name="storeName">नाम</string>
+    <string name="app_name">कैटीमा — वफादारी कार्ड वॉलेट</string>
```

Comment:
That's not the correct translation of Catima, it's way too long to fit in the title bar on all phone screens.

Reply:
Sorry for the inconvenience Mam please can you provide the correct translation. Actually mam I've made changes according to our previous discussion However have a question. I've checked the app on my local machine, but I'm not sure how to confirm if the issue is resolved. Based on my observation, it appears that there is no language selection feature present.

Reply:
I don't speak Hindi so I wouldn't know the correct translation but "Catima - Loyalty Card Wallet" is not "Catima". That's not an app name, that's a full app store title.

There is a language selector in Settings and your phone should have a built in language selector?...

Reply:
Yes, mam, I'm familiar with Hindi. Our app's name is 'Catima,' and its Hindi pronunciation is 'कैटीमा'.